substrait_consumer/functional/queries/sql/relations/read_relations.py

```diff
@@ -0,0 +1,36 @@
+from substrait_consumer.producers.duckdb_producer import DuckDBProducer
+from substrait_consumer.producers.datafusion_producer import DataFusionProducer
+from substrait_consumer.producers.isthmus_producer import IsthmusProducer
+
+READ_RELATIONS = {
+    "read_named_table": (
+        """
+        SELECT PS_PARTKEY FROM '{}'
+        """,
+        [DuckDBProducer, DataFusionProducer, IsthmusProducer],
+    ),
+    "isthmus_read_virtual_table": (
+        """
+        SELECT 10
+        """,
+        [IsthmusProducer],
+    ),
+    "datafusion_read_virtual_table": (
+        """
+        CREATE TABLE IF NOT EXISTS valuetable AS VALUES(1,'HELLO'),(12,'DATAFUSION');
+        """,
+        [DataFusionProducer],
+    ),
+    "duckdb_read_virtual_table": (
+        """
+        CREATE TABLE IF NOT EXISTS t1 (i INTEGER, j INTEGER);
```

Comment:
@EpsilonPrime 
```
E           duckdb.duckdb.InternalException: INTERNAL Error: Root node has more than 1, or 0 children (0) up to reaching a projection node. Type 126
E           This error signals an assertion failure within DuckDB
```

This should be fixed in the next release right?  It's what happens when trying to create the substrait plan.  Error is in DuckDB.

Reply:
I believe so.